colossalai/communication/p2p.py

```diff
@@ -19,12 +19,12 @@
 def _get_tensor_shape(tensor_shape: TensorShape, chunk_tensor: bool = False) -> Tuple[TensorShape, bool]:
     """get the exact tensor shape when communicating and return whether the tensor is a chunk
```

Comment:
This is exactly a helper function. No need for docstring for this function.

Reply:
This docstring would not appear in our documentation because this is a helper function. However, I change the style for all docstring for consistency.